server/trillian_log_server/main.go

```diff
@@ -149,7 +149,7 @@ func main() {
 	go awaitSignal(rpcServer)
 
 	if err := rpcServer.Serve(lis); err != nil {
-		glog.Fatalf("RPC server terminated on port %d: %v", *serverPortFlag, err)
+		glog.Errorf("RPC server terminated on port %d: %v", *serverPortFlag, err)
```

Comment:
I meant this one to be `Errorf`, so the log and map servers are more similar.  It also means the code below runs on shutdown.